pyramid_openapi3/__init__.py

```diff
@@ -59,6 +59,7 @@ def wrapper_view(context, request):
                     )
                 else:
                     # Do the view
+                    request.openapi_validated.raise_for_errors()
```

Comment:
If the `_pyramid_openapi3_validation_view_name` view is not configured, we shouldn't just silently swallow validation errors and continue, but raise the appropriate validation exception.